core/web/evm_transactions_controller.go

```diff
@@ -47,3 +60,131 @@ func (tc *TransactionsController) Show(c *gin.Context) {
+		KeyStore:    app.GetKeyStore().Eth(),
+	}
+}
+
+// Create signs and sends transaction from specified address. If address is not provided uses one of enabled keys for
+// specified chain.
+func (tc *EvmTransactionController) Create(c *gin.Context) {
+	if !tc.Enabled {
+		jsonAPIError(c, http.StatusUnprocessableEntity, errors.New("transactions creation disabled. To enable set TxmAsService.Enabled=true"))
+		return
+	}
+	var tx models.CreateEVMTransactionRequest
+	if err := c.ShouldBindJSON(&tx); err != nil {
+		jsonAPIError(c, http.StatusBadRequest, err)
+		return
+	}
+
+	if tx.IdempotencyKey == "" {
+		jsonAPIError(c, http.StatusBadRequest, errors.New("idempotencyKey must be set"))
+		return
+	}
+
+	decoded, err := hexutil.Decode(tx.EncodedPayload)
+	if err != nil {
+		jsonAPIError(c, http.StatusBadRequest, errors.Errorf("encodedPayload is malformed: %v", err))
+		return
+	}
+
+	if tx.ChainID == nil {
+		jsonAPIError(c, http.StatusBadRequest, errors.New("chainID must be set"))
+		return
+	}
+
+	if tx.ToAddress == nil {
+		jsonAPIError(c, http.StatusBadRequest, errors.New("toAddress must be set"))
+		return
+	}
+
+	chain, err := getChain(tc.Chains, tx.ChainID.String())
+	if err != nil {
+		if errors.Is(err, ErrMissingChainID) {
+			jsonAPIError(c, http.StatusUnprocessableEntity, err)
+			return
+		}
+
+		tc.Logger.Errorf("Failed to get chain", "err", err)
+		jsonAPIError(c, http.StatusInternalServerError, err)
+		return
+	}
+
+	if tx.FromAddress == utils.ZeroAddress {
+		tx.FromAddress, err = tc.KeyStore.GetRoundRobinAddress(tx.ChainID.ToInt())
+		if err != nil {
+			jsonAPIError(c, http.StatusUnprocessableEntity, errors.Errorf("failed to get fromAddress: %v", err))
+			return
+		}
+	} else {
+		_, err = tc.KeyStore.GetRoundRobinAddress(tx.ChainID.ToInt(), tx.FromAddress)
+		if err != nil {
+			jsonAPIError(c, http.StatusUnprocessableEntity,
+				errors.Errorf("fromAddress %v is not available: %v", tx.FromAddress, err))
+			return
+		}
+	}
+
+	if tx.FeeLimit == 0 {
+		tx.FeeLimit = chain.Config().EVM().GasEstimator().LimitDefault()
+	}
+
+	value := tx.Value.ToInt()
+	if value == nil {
+		value = big.NewInt(0)
+	}
+	etx, err := chain.TxManager().CreateTransaction(c, txmgr.TxRequest{
+		IdempotencyKey:   &tx.IdempotencyKey,
+		FromAddress:      tx.FromAddress,
+		ToAddress:        *tx.ToAddress,
+		EncodedPayload:   decoded,
+		Value:            *value,
+		FeeLimit:         tx.FeeLimit,
+		ForwarderAddress: tx.ForwarderAddress,
+		Strategy:         txmgrcommon.NewSendEveryStrategy(),
+	})
+	if err != nil {
+		jsonAPIError(c, http.StatusBadRequest, errors.Errorf("transaction failed: %v", err))
+		return
+	}
+
+	tc.AuditLogger.Audit(audit.EthTransactionCreated, map[string]interface{}{
+		"ethTX": etx,
+	})
+
+	// skip waiting for txmgr to create TxAttempt
+	if tx.SkipWaitTxAttempt {
+		jsonAPIResponse(c, presenters.NewEthTxResource(etx), "eth_tx")
+		return
+	}
+
+	timeout := 10 * time.Second // default
```

Comment:
As commented above, let us remove the lines 178-189.
We can directly return the Tx